build.rs

```diff
@@ -1,11 +1,13 @@
 use std::fs;
 
-use clap::Shell;
+use clap_complete::{generate_to, Shell};
+use Shell::*;
+//use clap_complete::shells::Shel{Bash, Fish, PowerShell, Elvish};
 
 include!("src/app.rs");
 
 fn main() {
-    let min_version = "1.53";
```

Comment:
Can we please also update all `README.md` files to include the new version?

Reply:
done